day_7/high_level.md

```diff
@@ -0,0 +1,15 @@
+# Task:  
+**Write up high level notes about how you would solve one of the following problems.**  
+
+**Chosen Problem:** Checker board  
+
+1. Establish `var` for board size user input. In the example, this is 6, so we'll use that first to make sure we're on the right track. Note that this will indicate both the character length of the row as well as the number of rows.  
+2. Establish `var` for the board (this is what you'll ultimately console log). Needs to be an empty string because you'll assign either black (X) or white (" ")strings to this.  
+3. Next, we need to establish a way to log each row and also log each element that goes in each row (` ` or `X`). It's clear we'll need to apply loops in some capacity.  
+4. Upon some Googling, "nested" for loops seem to be the answer. The website CodingRooms says: "The outer loop always executes first, and the inner loop executes inside the outer loop each time the outer loop executes once." That's what we need! The "outer loop" will execute each row, and the "inner loop" will execute what shows up in each row.  
```

Comment:
Nice work researching this problem and clearly writing out the steps you'd take to solve it!